my-app/src/components/login/login.js

```diff
@@ -1,112 +1,89 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable react/jsx-filename-extension */
 import React from 'react';
 import '../../app.css';
-import {Link, withRouter} from "react-router-dom";
-import axios from 'axios';
-
+import { Link, withRouter } from 'react-router-dom';
 
 function validate(username, password) {
-    const errors = [];
-  
-    if (username.length < 6) {
-        errors.push("Username should be at least 5 characters long");
-      }
-    if (password.length < 6) {
-      errors.push("Password should be at least 5 characters long");
-    }
-    return errors;
+  const errors = [];
+
+  if (username.length <= 1) {
+    errors.push('Please enter a username');
   }
+  if (password.length <= 1) {
+    errors.push('Please enter a password');
+  }
+  return errors;
+}
 
 class Login extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-              username: " ",
-              password: " ", 
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: ' ',
```

Comment:
It was a good idea initializing the username and password to an empty string instead of something like null or undefined.